Exercise_C.js

```diff
@@ -15,4 +15,13 @@ See README.md if you need help running your code.
 
 for (let i = 1; i <= 30; i++) {
   // CODE HERE
-}
+ if (i%3===0 && i%5===0) {
+  console.log("Hong Kong");
+
+} else if (i%3===0) {
+  console.log("Hong");
+
+} else if (i%5===0) {
+  console.log("Kong");
+ };
+};

```

Comment:
This code is almost correct, the only thing that was left out is that you did not add a condition for when the value is not divisible by 3 & 5 and also 3 or 5. But then if the number cannot be divided, we have to return the number. The code that would need to be added is ` }else{
   console.log(i);
 }`